pages/_app.tsx

```diff
@@ -108,21 +109,23 @@ function MyApp({ Component, pageProps }: AppProps) {
               autoConnect
             >
               <StarknetIdJsProvider>
-                <ThemeProvider theme={theme}>
-                  <Head>
-                    <title>Starknet.id</title>
-                    <meta
-                      name="viewport"
-                      content="width=device-width, initial-scale=1"
-                    />
-                  </Head>
-                  <Navbar />
-                  <AcceptCookies message="We'd love to count you on our traffic stats to ensure you get the best experience on our website !" />
-                  <PostHogProvider client={posthog}>
-                    <Component {...pageProps} />
-                  </PostHogProvider>
-                </ThemeProvider>
-                <Analytics />
+                <FormProvider>
```

Comment:
Could you add the form provider only where it's needed and not at the route of the app ? Imo we need the context only for the register/renew so we could have one context on these pages and not on all the app. 

Wdyt ?

GPT comments

```
Adding FormProvider More Precisely
If your form usage is more localized or you prefer to encapsulate form state within specific parts of your application, you might consider adding the FormProvider closer to where it's needed, such as:

Within specific page components: If only certain pages use forms, you can wrap those page components with the FormProvider to limit context scope and potentially improve performance.
Within a specific form-heavy section or layout: If a segment of your application, like a multi-step form wizard or a specific layout that includes multiple forms, is the primary user of form functionality, you might encapsulate the FormProvider there.
```

Reply:
yes I moved it to the register and renewal pages only